static-site/src/sections/sars-cov-2-forecasts-page.jsx

```diff
@@ -40,6 +40,29 @@ const introContents = [
   },
 ];
 
+
+const nextstrainCladesSrcBase = "https://nextstrain-data.s3.amazonaws.com/files/workflows/forecasts-ncov/gisaid/trial/viz-library-mk2/nextstrain_clades/global";
```

Comment:
Need to update this URL once the latest forecasts figures have been uploaded to S3

Reply:
Updated in force-push.